pom.xml

```diff
@@ -0,0 +1,134 @@
+<!--
+
+    Copyright (C) 2010 FuseSource, Corp. All rights reserved.
+    http://fusesource.com
+
+    The software in this package is published under the terms of the AGPL license
+    a copy of which has been included with this distribution in the license.txt file.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+ 
+  <parent>
+    <groupId>org.jboss.tools</groupId>
+    <artifactId>parent</artifactId>
+    <version>4.4.0.Alpha1-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.jboss.tools</groupId>
+  <artifactId>fuse-extras</artifactId>
+  <name>JBoss Tools Fuse Extras</name>
+  <version>1.0.0-SNAPSHOT</version>
```

Comment:
I am rather unsure here. Is everything we will hold in this repo used for fuse tooling? I wanted to keep a little separation between the reactor and the child projects.